contribute/README.md

```diff
@@ -11,22 +11,22 @@ coverY: 0
 
 ### Report a bug 🐞
 
-Do you think you see a bug in our code? To report a bug that _isn't a vulnerability_, go to the relevant GitHub repository, click on the _Issues_ tab, and select _Bug Report_.
+Do you think you see a bug in the code? To report a bug that _isn't a vulnerability_, go to the relevant GitHub repository, click on the _Issues_ tab, and select _Bug Report_.
```

Comment:
maybe it's just me, and it's def nitpicking, `Do you think you see a bug in the code` sound weird 😆 . Does `Found a bug in the code?` sound better?